opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/service/CiModuleService.java

```diff
@@ -0,0 +1,65 @@
+package nl.novi.opdrachttechiteasy.service;
+import jakarta.persistence.EntityNotFoundException;
+import nl.novi.opdrachttechiteasy.exceptions.RecordNotFoundException;
+import nl.novi.opdrachttechiteasy.models.CiModule;
+import nl.novi.opdrachttechiteasy.repositories.CiModuleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+
+@Service
+public class CiModuleService {
+
+    @Autowired
```

Comment:
De `@`Autowired annotatie is hier niet nodig, omdat je het via de constructor injecteert. De annotatie is nodig wanneer je het de variabele zonder constructor wilt injecteren. Dat wordt echter afgeraden